io.catenax.specialCharacteristics.measurement/1.0.0/Measurement.ttl

```diff
@@ -0,0 +1,63 @@
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.specialCharacteristics.measurement:1.0.0#> .
+@prefix bpn: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+@prefix samm-u: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix serialPart: <urn:samm:io.catenax.serial_part:3.0.0#> .
+
+:Measurement a samm:Aspect ;
```

Comment:
Is there a reason why you name the aspect model _'Measurement'_? If you want to create an aspect model for special characteristics does it not make more sense to name the aspect SpecialCharacteristic or SpecialCharacteristicMeasurment?

Reply:
No, the idea was to show that it is related to specialCharacteristics through the path in the name.

**io.catenax.specialCharacteristics.measurements**/1.0.0/

I guess this should not be a serious problem in renaming the model

Reply:
renamed it to specialCharacteristicMeasurement